src/components/navigation/Nav.js

```diff
@@ -1,5 +1,7 @@
 import React, { useState } from 'react';
 import { NavLink } from 'react-router-dom';
+import { AiOutlineClose, AiOutlineMenu } from 'react-icons/ai';
+import logo from '../../assets/logo.png';
```

Comment:
- [ ] There doesn't seem to be a logo in the repository.  The directory `../../assets` doesn't seem to be in the resositoty. Maybe check again, and push it.